src/commands/junit/renderer.ts

```diff
@@ -0,0 +1,43 @@
+import chalk from 'chalk'
+
+import {Payload} from './interfaces'
+
+const ICONS = {
+  FAILED: chalk.bold.red('❌'),
+  SUCCESS: chalk.bold.green('✅'),
+  WARNING: chalk.bold.green('⚠️'),
```

Comment:
Do you need to wrap with chalk here?
```suggestion
  FAILED: '❌',
  SUCCESS: '✅',
  WARNING: '⚠️',
```

Reply:
probably not. This is copy pasted from `sourcemaps`. I'll change it 👍